play.html

```diff
@@ -13,6 +13,7 @@
 	<script src="scripts/play/bot.js"></script>
```

Comment:
https://github.com/CarbonChess/CarbonChess/blob/2fdd9a79b1ebbb18a723dc657d3b3c73dc83c094/play.html#L39

Todo: change to use sendDB()